client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -765,6 +777,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 					screenshot={ fullLengthScreenshot }
 					isExternallyManaged={ selectedDesign.is_externally_managed }
 					isVirtual={ selectedDesign.is_virtual }
+					disableGlobalStyles={ disableCheckoutImmediately }
```

Comment:
> I just want to use another name to make it clear when people go through the codebase. When I work on this feature initially, I could not understand why we disabled premium themes and global styles in the design-first flow 🥲

I see, understandable. What about this one? I think this is redundant as we already disabled Premium / style variations 😄

Reply:
We need this to disable Colors & Fonts in the design-first flow. Actually, it's buggy now as people can pick Colors & Fonts and upgrade immediately in the design-first flow. But when they are redirected back to the Launchpad, the task, Pick a plan, won't be completed 🫠

Reply:
Ohhh, I see!! 😂